examples/machine-learning/a4x-highgpu-4g/a4xhigh-slurm-blueprint.yaml

```diff
@@ -29,7 +29,7 @@ vars:
   image_build_disk_type: hyperdisk-balanced
   image_disk_size_gb: 100
   built_image_family: slurm-ubuntu2404-accelerator-arm64-64k
-  build_from_git_ref: 6.10.5
+  build_slurm_from_git_ref: 6.10.6
```

Comment:
You have to update where `build_from_git_ref` used to be referenced as well later on down the blueprint